tests/async_client_test.py

```diff
@@ -152,6 +152,7 @@
 BASE_LEGACY_MEDIA_URL = f"https://example.org{MATRIX_LEGACY_MEDIA_API_PATH}"
 TEST_ROOM_ID = "!testroom:example.org"
 TEST_ROOM_ID2 = "!testroom2:example.org"
+TEST_ROOM_ID_V12 = "$5hdALbO+xIhzcLTxCkspx5uqry9wO8322h/OI9ApnHE"
```

Comment:
```suggestion
TEST_ROOM_ID_V12 = "!5hdALbO+xIhzcLTxCkspx5uqry9wO8322h/OI9ApnHE"
```

[See MSC4291](https://github.com/matrix-org/matrix-spec-proposals/blob/main/proposals/4291-room-ids-as-hashes.md)

Reply:
Oops.